main.go

```diff
@@ -11,15 +11,19 @@ import (
 
 const (
 	version  = "0.0.1"
-	logLevel = "debug"
+	logLevel = "info"
 )
 
+var debug = false
```

Comment:
the usage of a global variable can be avoided, see the comment below on the method "addBefore"